src/main/sensors/acceleration.c

```diff
@@ -46,6 +47,27 @@
 
 PG_REGISTER_PROFILE_WITH_RESET_FN(accelerometerConfig_t, accelerometerConfig, PG_ACCELEROMETER_CONFIG, 0);
 
+int32_t accADC[XYZ_AXIS_COUNT];
+int32_t accSmooth[XYZ_AXIS_COUNT];
+
+acc_t acc;                       // acc access functions
+sensor_align_e accAlign = 0;
+uint32_t accTargetLooptime;
+
+uint16_t calibratingA = 0;      // the calibration is done is the main loop. Calibrating decreases at each cycle down to 0, then we enter in a normal mode.
+
+extern uint16_t InflightcalibratingA;
```

Comment:
This `extern` in .c file is so bad idea. It's only matter of time it will cause hard-to-find bug ... 
Move the externs to some header files (even creating one if necessary)..